community/planning/features.md

```diff
@@ -0,0 +1,24 @@
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+# Feature Planning
+
+Features listed here exist in various stages of the development process.  Many
+are in the design phase and may result in significant changes that may impact
+applications built on top of Splinter and its constellation of related libraries
+and services.
+
+We welcome discussion on these topics, either on Slack or during the engineering
+forum.
```

Comment:
Might want to add some links here or refer back to the root /community page (which should probably have a link in the left sidebar too).

Reply:
Is there an example of this?

Reply:
Looks like all the Community pages need this.  Will do in a separate PR

Reply:
Just seems like this should provide a link to the Slack and the engineering forum. Someone reading this may not know where to find them. This can be addressed in a future PR.

Reply:
Those items are in the main community section already.  Probably we need to figure out a common strategy for back links to top-level sections from nested sections like these.